tests/e2eTest/v0.3.4.ts

```diff
@@ -39,7 +39,7 @@ const test_suit = (setTezosToolkit: (tezos: TezosToolkit) => TezosToolkit) =>
         expect(storage.proposal_counter.isEqualTo(BigNumber(0))).toBe(true);
         expect(storage.owners).toEqual([owner]);
         storage.metadata.get("").then((value: string) => {
-          expect(value).toEqual(char2Bytes(ipfs_file));
+          expect(value).toEqual(stringToBytes(ipfs_file));
```

Comment:
what is the difference between `char2Bytes` and `stringToBytes` ? Why do we need to change it?

Reply:
`charToBytes` and `bytesToChar` are deprecated => https://taquito.io/typedoc/functions/_taquito_utils.char2Bytes